server/aws-lsp-codewhisperer/src/shared/amazonQServer.ts

```diff
@@ -35,6 +38,16 @@ export const AmazonQServiceServerFactory =
                 sdkInitializator,
             })
 
+            // Initialize ATX Token Service Manager for ATX FES support
+            AtxTokenServiceManager.initInstance({
```

Comment:
feels like the atx management should be in its own server factory? or is it still too deeply intertwined with the existing q service?